app.py

```diff
@@ -12,25 +12,29 @@
 
 @app.route('/', methods=['post', 'get'])
 def login():
-	location = ''
 	if request.method == 'POST':
 		location = request.form.get('address/location')
-		img_bytes = get_image(location)
-		image = add_bernie(img_bytes)
+		if location.strip() != '':
```

Comment:
This fails with AttributeError: 'NoneType' object has no attribute 'strip'. Can we also check for NoneType before the strip condition? Something like _if location is not None and location.string.strip() != '':_

Reply:
looks like empty search was handled in another commit, but I'm using your suggestion to handle whitespace input! thanks!